Appium/entry_point.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+JSON="/root/nodeconfig.json"
```

Comment:
Sure. Will rename it. I'm still looking in to making a better test for this functionality using [mocks](https://github.com/jasonkarns/bats-mock) but have some problems getting the right output from the mocking lib.